src/features/newTab/default/topSites/index.ts

```diff
@@ -97,7 +96,6 @@ export const Tile = styled<TileProps, 'div'>('div')`
 `
 
 export const TileFavicon = styled<{}, 'img'>('img')`
-  box-sizing: border-box;
   display: block;
-  height: 64px;
+  height: 40px;
 `
```

Comment:
@cezaraugusto just wanted to bring attention to the size update for the favicons here. i noticed some updates regarding favicon sizing in the Core PR and wasn't sure if this would mess it up.

Reply:
ya I updated brave-core to fetch 64px favicons as well